packages/twenty-server/src/engine/core-modules/billing/dtos/billing-subscription-schedule-phase.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+export class BillingSubscriptionSchedulePhaseItem {
```

Comment:
OMG, this looks like a new layer of complexity :D

Reply:
Yep. It's necessary to schedule billing downgrade